packages/migration/src/reindex-search.ts

```diff
@@ -25,46 +25,13 @@ const triggerReindex = async () => {
     )
   }
 
-  const data = await response.json()
-  return data.jobId
-}
-
-/**
- * Checks the status of the reindex, as it can take a while
- */
-const checkReindexStatus = async (jobId: string) => {
-  const response = await fetch(new URL(`reindex/status/${jobId}`, SEARCH_URL), {
-    method: 'GET'
-  })
-
-  if (!response.ok) {
-    throw new Error(
-      `Problem checking reindex status from ElasticSearch. Response: ${await response.text()}`
-    )
-  }
-
-  const data = await response.json()
-  return data.status === 'completed'
+  return response
 }
 
 async function main() {
   console.info(`Reindexing search...`)
-  const jobId = await triggerReindex()
-  await new Promise<void>((resolve, reject) => {
-    const intervalId = setInterval(async () => {
-      try {
-        const isCompleted = await checkReindexStatus(jobId)
-        if (isCompleted) {
-          clearInterval(intervalId)
-          resolve()
-        }
-      } catch (error) {
-        clearInterval(intervalId)
-        reject(error)
-      }
-    }, 1000)
-  })
-  console.info(`...done reindexing search with job id ${jobId}`)
+  await triggerReindex()
```

Comment:
Won't this take quite a bit to finish when there are lots of records. Are we planning to do something similar to v1 at a later time?